docs/index.rst

```diff
@@ -41,3 +41,11 @@ Command line interface
 
    self
    changelog
+
+Python API
```

Comment:
Here I took the same approach I saw for the documented CLI.
Please let me know if you prefer something more verbore/descriptive.